wp-content/themes/citylimits/less/_navbar.less

```diff
@@ -290,4 +290,53 @@ html.nav-open body {
     @media (min-width: 769px) {
         display: none;
     }
+}
+#secondary-nav {
+    .secondary-special-projects-nav {
+        list-style: none;
+        margin-left: 0;
+        margin-top: -5px;
+        display: flex;
+        align-items: center;
+        background: @red;
+        li {
+            flex-grow: 1;
+            flex-basis: 0;
+            margin: 20px 0;
+            border-right: 1px solid @grey1;
+            padding-left: 5%;
```

Comment:
Should this padding be replicated on the right? As it stands, it looks kinda tossed-together: ![Screen Shot 2019-10-22 at 18 26 41 ](https://user-images.githubusercontent.com/1754187/67339617-911ad680-f4f9-11e9-97bf-654a42e29005.png)


Should it be reduced to `@measure`? In the below screenshot, `@measure`'s `1.5rem` replaces the `2%`.

![Screen Shot 2019-10-22 at 18 28 39 ](https://user-images.githubusercontent.com/1754187/67339753-ccb5a080-f4f9-11e9-8aec-7d268f1799e1.png)

Reply:
@benlk https://github.com/INN/umbrella-citylimits/pull/78/commits/79610c5e9b67fe13113c273e46d440073db9b52d uses `@measure` for the left padding and adds it to the right as well